sources/packages/backend/apps/api/src/route-controllers/application-change-request/_tests_/e2e/application-change-request.aest.controller.pendingApplicationChangeRequest.e2e-spec.ts

```diff
@@ -0,0 +1,336 @@
+import { HttpStatus, INestApplication } from "@nestjs/common";
+import * as request from "supertest";
+import { DataSource } from "typeorm";
+import {
+  AESTGroups,
+  BEARER_AUTH_TYPE,
+  createTestingAppModule,
+  getAESTToken,
+} from "../../../../testHelpers";
+import { ApplicationEditStatus, ApplicationStatus } from "@sims/sims-db";
+import {
+  createE2EDataSources,
+  E2EDataSources,
+  saveFakeApplication,
+} from "@sims/test-utils";
+import MockDate from "mockdate";
+
+describe("ApplicationChangeRequestAESTController(e2e)-pendingApplicationChangeRequest", () => {
+  let app: INestApplication;
+  let appDataSource: DataSource;
+  let db: E2EDataSources;
+
+  beforeAll(async () => {
+    const { nestApplication, dataSource } = await createTestingAppModule();
+    app = nestApplication;
+    appDataSource = dataSource;
+    db = createE2EDataSources(dataSource);
+  });
+
+  beforeEach(async () => {
+    MockDate.reset();
+  });
+
+  it("Should successfully retrieve paginated list of pending application change requests for an AEST user", async () => {
```

Comment:
Can we remove this test scenario. As it does not validate anything precisely, I do not see this test creating much value.